modules/resetdigitalBidAdapter.js

```diff
@@ -43,16 +43,29 @@ export const spec = {
       };
     }
 
+    if (bidderRequest && bidderRequest.uspConsent) {
+      payload.ccpa = bidderRequest.uspConsent;
+    }
+
     for (let x = 0; x < validBidRequests.length; x++) {
       let req = validBidRequests[x]
 
       payload.imps.push({
         pub_id: req.params.pubId,
+        site_id: req.params.siteID ? req.params.siteID : null,
+        placement_id: req.params.placement ? req.params.placement : null,
+        position: req.params.position ? req.params.position : null,
+        bid_floor: req.params.bidFloor ? req.params.bidFloor : null,
+        lat_long: req.params.latLong ? req.params.latLong : null,
+        inventory: req.params.inventory ? req.params.inventory : null,
+        visitor: req.params.visitor ? req.params.visitor : null,
+        keywords: req.params.keywords ? req.params.keywords : null,
```

Comment:
this is also true about keywords, the bid request object should have keywords you can append to the keywords you see in params

Reply:
I reviewed other adapters and couldn't find one that takes the keywords from the bid request object, so I think we are covered about that. in another case can you share an example with me?